.github/workflows/readme-changelog-sync.yml

```diff
@@ -0,0 +1,57 @@
+name: Sync Changelogs to ReadMe
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    paths:
+      - 'changelog.json'
+      - 'changelog_index.json'
```

Comment:
can probably drop `changelog_index` and `operation_ids` from here and just be `changelog.json`